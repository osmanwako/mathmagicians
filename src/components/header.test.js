import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Header from './header';

test('Header component renders correctly', () => {
    const tree = renderer.create(
        <MemoryRouter>
            <Header />
        </MemoryRouter>
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
