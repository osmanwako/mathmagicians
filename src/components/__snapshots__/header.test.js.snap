// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders correctly 1`] = `
<nav
  className="nav-header"
>
  <h1
    className="nav-logo"
  >
    Math Magicians 
  </h1>
  <ul
    className="nav-unorder"
  >
    <li
      className="nav-list"
    >
      <a
        className="nav-link"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        className="nav-link"
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        className="nav-link link-quote"
        href="/quotes"
        onClick={[Function]}
      >
        Quotes
      </a>
    </li>
  </ul>
</nav>
`;
